postinstall.js

```diff
@@ -71,8 +72,21 @@ function hasCache() {
 function download(cb) {
 	console.log('ngrok - downloading binary ' + cdnUrl);
 
+	const caString = execSync('npm config get ca')
```

Comment:
will execSync throw an exception if "npm" binary is not found?
if yes, please wrap it into the try-catch

Reply:
Checking now.

Reply:
Yes, but if we use the environment variable, this concern is solved, so I'll do that. Assuming the path specified is invalid, do you want to attempt without a CA or crash?

Reply:
just check if var is present and then try-catch extracting the CA from the path

Reply:
Added this code. If the path is undefined, fs.existsSync will just return false.